### PR TITLE
Ac 559

### PR DESCRIPTION
**Pull request by dmotan**
### Checklist for Pull Requests

🚨Please review the [guidelines for contributing](./CONTRIBUTING.md) to this repository.

- [ ] Make sure your branch matches the naming conventions. (Something like: `AC-152`). Don't request your master!
- [ ] Your branch should only contain changes relevant to the ticket it is named after.
- [ ] Make sure you are making a pull request against the **dev branch** (left side). Also you should start _your branch_ off _our dev_.
- [ ] Check your code additions will fail neither code linting checks nor e2e test (this will be tested automatically when you open your PR).

### Description

Please describe anything in your pull request that isn't immediately understandable from the ticket description.

Thank you! 😎

### CODE REVIEW

File: src/styles/_city-page.scss [hunk 1]
```diff
@@ -8,6 +8,7 @@
 
   &__row {
     height: 28px;
+    width: 100%;;
```
Comment (bparrish17) [inline src/styles/_city-page.scss:11]:
```suggestion
    width: 100%;
```

File: src/styles/_shared.scss [hunk 2]
```diff
@@ -79,5 +79,12 @@ $colors: (
   a {
     color: white;
     font-size: 16px;
+    @media screen and (min-width: 1440px) {
```
Comment (bparrish17) [inline src/styles/_shared.scss:82]: create scss variable for screen size, see $md

File: src/pages/city.page.js [hunk 3]
```diff
@@ -103,7 +103,7 @@ class CityPage extends Component {
         <Header enableBackBtn={true}></Header>
         <Container className="content">
           <h1>
-            {this.state.cityName}, {this.state.stateName} COVID-19 Testing Centers ({get(this, 'state.centerList.length', 0)}) | AllClear
+            {this.state.cityName}, {this.state.stateName} COVID-19 Testing Centers ({get(this, 'state.centerList.length', 0)})
```
Comment (crimsonsunset) [inline src/pages/city.page.js:106]: Leave `| AllClear` part of titles

File: src/pages/state-list.page.js [hunk 4]
```diff
@@ -66,15 +66,15 @@ class StateListPage extends Component {
       <div className="tracing">
         <Header enableBackBtn={true}></Header>
         <Container className="content">
-          <h1>COVID-19 Testing Centers by State | AllClear</h1>
+          <h1>COVID-19 Testing Centers by State</h1>
```
Comment (crimsonsunset) [inline src/pages/state-list.page.js:69]: same

File: src/pages/state.page.js [hunk 5]
```diff
@@ -71,23 +71,23 @@ class StatePage extends Component {
       <div className="tracing">
         <Header enableBackBtn={true}></Header>
         <Container className="content">
-          <h1>{startCase(this.state.stateName)} COVID-19 Testing Centers ({get(this, 'state.numFacilities', 0)}) | AllClear</h1>
+          <h1>{startCase(this.state.stateName)} COVID-19 Testing Centers ({get(this, 'state.numFacilities', 0)})</h1>
```
Comment (crimsonsunset) [inline src/pages/state.page.js:74]: same

File: src/styles/_shared.scss (hunk 2 above)
Comment (crimsonsunset) [reply src/styles/_shared.scss:82]: yeah, $lg might be good enough in _variable.scss otherwise just call it something like $larger

Comment (crimsonsunset) [reply src/styles/_shared.scss:82]: could you also break this out into a seperate scss file? see app.scss and just add a new file in styles folder following the same convention

**Comment by crimsonsunset** [comment]
this layout should also be on the city list page and state list page

File: src/styles/_shared.scss (hunk 2 above)
Comment (dmotan) [reply src/styles/_shared.scss:82]: `.seo-list` is a shared class between multiple files and I believe that's why it's there from the beginning, I can split it into it's own file, but keep them as they are now will reduce redundancy. What do you think?

Comment (crimsonsunset) [reply src/styles/_shared.scss]: ah right, i guess theres not much individual styling so probably some kind of scss mixin would work or just leaving it the way it is. i dont feel strongly enough to make you change it again so ill just merge it

**Pull request merged**